src/main/server.js

```diff
@@ -248,12 +306,16 @@ function createServer(options) {
     //       answer: str,
     //       distractors: [str]
     //     }]
+    //     numVersions: num,
+    //     examReason: str
```

Comment:
@OfekN11 @MayElnathan

Reply:
having this documentation here isn't ideal as this area isn't really viewed/worked with.
moving it to the facade would be a better choice
and you can document only the returned value, and have the sent data be self documented with the call to validateParameters

Reply:
Agreed, will be moved